LEAF_Request_Portal/sources/Workflow.php

```diff
@@ -71,10 +71,8 @@ public function getSteps()
         return $out;
     }
 
-    public function deleteStep($stepID)
-    {
-        if (!$this->login->checkGroup(1))
-        {
+    public function deleteStep($stepID){
```

Comment:
```suggestion
    public function deleteStep(int $stepID): int|string {
```

Reply:
Great addition, I tested it and it works well! thank you.